### PR TITLE
Display initialization message

### LINKED ISSUE #8525: Display initialization message
As in the old GUI, we should display some message informing the user that the project is still opening. The message should disappear after first execution completion.

### PR DESCRIPTION
**Pull request by MichaelMauderer**
### Pull Request Description

Implements #8525

[Screencast from 18-12-23 14:40:29.webm](https://github.com/enso-org/enso/assets/1428930/c7db2a9d-0020-4bce-b44c-d6712df58098)

### Important Notes

Previously, we re-used the API from the dashboard. But this mechanism is brittle and is currently even broken with the new GUI. So instead, I just used the Vue equivalent toast API, which seems easier to use.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] Unit tests have been written where possible.
  - [x] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

**Review by somebody1234 — APPROVED** [review]
no major issues, but the primary issue that may prevent this from being a long term solution, is that if it doesn't properly integrate with its reaactcounterpart, vertical stacking of multiple toasts probably doesn't work properly

File: app/gui2/src/components/GraphEditor.vue [hunk 1]
```diff
@@ -52,7 +52,7 @@ const suggestionDb = useSuggestionDbStore()
 const interaction = provideInteractionHandler()
 
 function initStartupToast() {
-  const startupToast = toast.info('Compiling standard library. It can take up to 1 minute.', {
+  const startupToast = toast.info('Initialising project, It can take up to one minute.', {
```
Comment (somebody1234) [inline app/gui2/src/components/GraphEditor.vue]: I think we prefer american english right? which would make it "Initializing"

**Comment by MichaelMauderer** [comment]
> no major issues, but the primary issue that may prevent this from being a long term solution, is that if it doesn't properly integrate with its reaactcounterpart, vertical stacking of multiple toasts probably doesn't work properly

This will probably be solved in the follow-up PR. Since we use the same underlying library, this should be compatible. I'm just now doing some follow up testing to check whether it behaves nicely.

**Review by Frizi — APPROVED** [review]
The interoperability with toasts from dashboard definitely needs to be solved, but this can be merged as is and it can be fixed in future PR.

**Pull request merged**